tests/ui/traits/ice-with-dyn-pointee-errors.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(ptr_metadata)]
```

Comment:
Could you please either reference https://github.com/rust-lang/rust/issues/112737 in a comment or change the filename to issue-112737.rs?

Reply:
I'd rather the issue referenced in a comment